packages/app-store/alby/api/add.ts

```diff
@@ -41,19 +40,3 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
   return res.status(200).json({ url: "/apps/alby/setup" });
 }
-/*import { createDefaultInstallation } from "@calcom/app-store/_utils/installation";
```

Comment:
Did we get an answer here? ideally there is no need for a custom handler

Reply:
My guess here is normal apps doesn't require a handler but since this is a payment and requires setup we should go for the custom one.